ci-operator/step-registry/openstack/conf/addetcdramdisk/openstack-conf-addetcdramdisk-ref.yaml

```diff
@@ -0,0 +1,18 @@
+ref:
+  as: openstack-conf-addetcdramdisk
+  from: installer
+  commands: openstack-conf-addetcdramdisk-commands.sh
+  env:
+    - name: SIZEOFRAMDISK
```

Comment:
This variable does not appear to have any effect. It is not used by the script.

Reply:
done